perception/autoware_tensorrt_bevdet/CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+cmake_minimum_required(VERSION 3.17)
+project(autoware_tensorrt_bevdet)
+
+add_compile_options(-W)
+add_compile_options(-std=c++17)
+set(CMAKE_CXX_FLAGS_RELEASE "-Wno-deprecated-declarations -O2")
+set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS} -std=c++17 -O0 -Xcompiler -fPIC -g -w -gencode=arch=compute_86,code=sm_86")
```

Comment:
Consider adding `-Wall` to `add_compile_options` for more comprehensive warnings during compilation. This helps catch potential issues early in the development process.  For example:  `add_compile_options(-Wall -Wextra -Wpedantic -Werror)`  (Adjust `-Werror` based on your build system's capabilities).